Content.Shared/Hands/EntitySystems/SharedHandsSystem.cs

```diff
@@ -77,6 +77,35 @@ public virtual void RemoveHand(EntityUid uid, string handName, HandsComponent? h
         Dirty(handsComp);
     }
 
+    /// <summary>
+    /// Gets rid of all the entity's hands. Also gets rid of HandsComponent.
+    /// </summary>
+    /// <param name="uid"></param>
+    /// <param name="handsComp"></param>
+
+    public void RemoveHands(EntityUid uid, HandsComponent? handsComp = null)
+    {
+        if (!Resolve(uid, ref handsComp))
+            return;
+
+        RemoveHands(uid, EnumerateHands(uid), handsComp);
+        RemComp(uid, handsComp);
```

Comment:
this should still be in zombie code. removing all of the hands shouldn't necessarily mean removing the component as well.

Reply:
Noted